scripts/tasks/ts.ts

```diff
@@ -40,10 +44,13 @@ function prepareTsTaskConfig(options: TscTaskOptions) {
     const normalizedOptions = { ...options };
     normalizedOptions.baseUrl = '.';
     normalizedOptions.rootDir = './src';
+    normalizedOptions.project = path.basename(tsConfigFilePath);
 
     return normalizedOptions;
   }
 
+  options.target = 'es5';
```

Comment:
you didn't remove any hardcoded `es5` targets right ? it seems rather that you added them

Reply:
nope. I removed them from all tsc tasks by default to maintain old behaviours. They are added to non converge/modern packages to not break any behaviours

Reply:
I actually just checked this recently, the hardcoded `es5` target is never needed because all v8-related packages explicitly declare a `target`. (That's why it was removed from this task by @GeoffCoxMSFT's recent PR to update libs/target for converged.)